Tests/BootCamp.Chapter.Tests/ConsoleTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.IO;
+using BootCamp.Chapter.Tests.Utils;
+
+namespace BootCamp.Chapter.Tests
+{
+    public class ConsoleTests: IDisposable
+    {
+        /// <summary>
+        /// Disposes console output file and returns its' contents.
+        /// </summary>
+        protected string ConsoleOutput
+        {
+            get
+            {
+                if (_consoleOutput == null)
+                {
+                    throw new ArgumentNullException(nameof(_consoleOutput));
+                }
+
+                _consoleOutput.Dispose();
+                var content = FakeConsole.ReadAllText(_testKey);
+
+                return content;
+            }
+        }
+
+        /// <summary>
+        /// Stubs <see cref="Console.ReadLine()"/> with the value set.
+        /// </summary>
+        protected string ConsoleInput
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'ConsoleInput' a static property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4884711)